src/core/types/store.js

```diff
@@ -0,0 +1,14 @@
+import type { ReactRouterHistoryType } from 'core/types/router';
+
+// This is a type function that takes a state type and returns a reducer
+// type, i.e. a function that accepts and returns the same state type.
+/* eslint-disable no-undef */
```

Comment:
You could try removing this to see what happens. It's some kind of eslint/flow bug.

Reply:
it does not work :( I tried and it complains about `AnyState` not being defined..